src/Sql/Insert.php

```diff
@@ -226,7 +226,7 @@ public function __set($name, $value)
      */
     public function __unset($name)
     {
-        if (!isset($this->columns[$name])) {
+        if (!$this->__isset($name)) {
```

Comment:
Use `array_key_exists()` here as well; this is a case where DRY is not better, but instead adds performance overhead and indirection.

Make the same change on the `__get()` method, too.